server.js

```diff
@@ -59,9 +59,68 @@ function commandRouter({ user_id, user_name, text }, response) {
             firebase.deleteUser(user_id);
             response.end(":'( Goodbye " + user_name + ". It's been real.");
             break;
+        case 'add':
+            let newChannel = text.split(' ')[1];
+            if (!newChannel) {
+                response.end(
+                    'You forgot to send me the channel name you would like to register for. (EG `general`)'
+                );
+                break;
+            }
+            if (newChannel.includes('<')) {
+                newChannel = newChannel.split('|')[1].replace('>', '');
```

Comment:
whats this for?

Reply:
If you tab autocomplete a channel name in slack it adds some strange formatting. Something like `<channel|engineering>`

Reply:
interesting